lib/nimble_options/docs.ex

```diff
@@ -71,7 +71,7 @@ defmodule NimbleOptions.Docs do
   end
 
   defp get_required_str(schema) do
-    schema[:required] && "Required."
+    (schema[:required] && "Required.") || nil
```

Comment:
Why this change? 🤔

Reply:
Hi! `Enum.join(parts, " - ")` returns `"false - ..."` when `schema[:required]` is `false`. This is because the struct contains a schema that was passed through the schema validation function and now contains `false` for undefined docs.